src/app/todo/components/todoItem/todo-item.component.ts

```diff
@@ -6,26 +6,24 @@ import { TaskService } from '../../services/todo.services';
 import { ITask } from '../../models/todo.interface';
 import { switchMap } from 'rxjs/operators';
 import { TaskApiService } from '../../services/task-api.service';
+import { AutoUnsub } from '../../../core/decorators/auto-unsub.decorator';
 
 @Component({
   selector: 'todo-item',
   standalone: true,
-  templateUrl: './todoItem.component.html',
-  styleUrl: './todoItem.component.scss',
+  templateUrl: './todo-item.component.html',
+  styleUrl: './todo-item.component.scss',
   imports: [CommonModule, RouterLink],
 })
+@AutoUnsub()
 export class TodoItemComponent implements OnInit {
   taskItem: Observable<ITask | null> = of(null);
```

Comment:
all observables sohuld have $ at the end of the variable name for example:
taksItem$: Observable<....>  = ...